bitnami/mariadb/values.yaml

```diff
@@ -1089,7 +1099,9 @@ metrics:
   ##     readOnlyRootFilesystem: true
   ##
   containerSecurityContext:
-    enabled: false
+    enabled: true
```

Comment:
Could you explain why you change this to true?

Reply:
Hi @Mauraza!

I guess this was set to `false`, because the `containerSecurityContext` did not include any default parameters so far. Since this PR adds defaults, it should also apply them by enabling the `containerSecurityContext`. That's why I changed to `enabled=true`.

Reply:
Same as with the other containers, defaulting to `privileged=false` and `allowPrivilegeEscalation=false` should not impact the container in any way (I would be very suspicious if a metrics container needed privileged execution 😉) and just implements common security best-practices.

Reply:
But you can keep enabled to false because if someone updates the chart to this version it will not change. 
Could you change it?

Reply:
@Mauraza when the chart is updated, this will only change if people stick to the defaults and that's what this PR is actually all about. It is not a breaking change. But sure, I can revert this to disabled by default